docs/DeveloperGuide.md

```diff
@@ -1007,13 +1007,13 @@ The overview of the design on schedule features can be found [here](#schedule-re
 #### Update Schedule
 A summary of the general procedure of updating a plan for a particular day to the schedule in WerkIt! is as follows:
 1. User enters the command `schedule /update <day number> <plan number>`.
-2. If there are no plan being scheduled for the day, a new Day object is created and stored in the application.
-   If there is an existing plan scheduled for that particular day, the Day object that had already been created,
+2. If there is no plan being scheduled for the day, a new Day object is created and stored in the application.
+   If there is an existing plan scheduled for that particular day, the `Day` object that has already been created,
```

Comment:
Perhaps the comma at the end of line 1011 can be omitted?

Reply:
thanks for spotting this mistake